content/en/docs/instrumentation/net/libraries.md

```diff
@@ -1,23 +1,28 @@
 ---
-title: Automatic Instrumentation
+title: Using instrumentation libraries
 weight: 3
-linkTitle: Automatic
+linkTitle: Libraries
 ---
 
-.NET supports automatic instrumentation with [instrumentation
-libraries](/docs/reference/specification/glossary/#instrumentation-library) that
-generate telemetry data for a particular instrumented library.
+The [automatic instrumentation](/docs/reference/specification/glossary/#automatic-instrumentation)
```

Comment:
I think this section might be better listed near the bottom in a "See Also" section or something like that. If we're to rename this to be about libraries, then having something that's _not_ a library be the first thing mentioned in the doc might be confusing.

Reply:
I will do it tomorrow. Do you have a good example that I could use a reference?

Reply:
Not really, sadly. Maybe a subsection that just says "Automatic Instrumentation" or "Automatic Instrumentation Without Libraries" or something like that? Would prefer something more specific than "See Also", but I'm not too opinionated on the naming

Reply:
I have decided the move it to "Next steps" as it looks like a pattern in the .NET docs

Addressed in https://github.com/open-telemetry/opentelemetry.io/pull/1565/commits/739e1298b8d9df6145d4dc33992f1b2f335c0e24